diff -urpN cloud-init-0.7.5.orig/cloudinit/config/cc_resizefs.py cloud-init-0.7.5/cloudinit/config/cc_resizefs.py
--- cloud-init-0.7.5.orig/cloudinit/config/cc_resizefs.py	2014-04-01 11:26:07.000000000 -0700
+++ cloud-init-0.7.5/cloudinit/config/cc_resizefs.py	2014-09-19 17:14:17.000000000 -0700
@@ -154,8 +154,8 @@ def handle(name, cfg, _cloud, log, args)
         # Fork to a child that will run
         # the resize command
         util.fork_cb(
-            util.log_time(logfunc=log.debug, msg="backgrounded Resizing",
-                func=do_resize, args=(resize_cmd, log)))
+            util.log_time, logfunc=log.debug, msg="backgrounded Resizing",
+            func=do_resize, args=(resize_cmd, log))
     else:
         util.log_time(logfunc=log.debug, msg="Resizing",
             func=do_resize, args=(resize_cmd, log))
diff -urpN cloud-init-0.7.5.orig/cloudinit/util.py cloud-init-0.7.5/cloudinit/util.py
--- cloud-init-0.7.5.orig/cloudinit/util.py	2014-04-01 11:26:07.000000000 -0700
+++ cloud-init-0.7.5/cloudinit/util.py	2014-09-19 17:13:24.000000000 -0700
@@ -193,11 +193,11 @@ def ExtendedTemporaryFile(**kwargs):
     return fh
 
 
-def fork_cb(child_cb, *args):
+def fork_cb(child_cb, *args, **kwargs):
     fid = os.fork()
     if fid == 0:
         try:
-            child_cb(*args)
+            child_cb(*args, **kwargs)
             os._exit(0)  # pylint: disable=W0212
         except:
             logexc(LOG, "Failed forking and calling callback %s",
